lib/modules/default.nix

```
@@ -1,7 +1,7 @@
 { lib }:
 
-{
+rec {
   keyboard = import ./keyboard.nix { inherit lib; };
-
-  themes = import ./themes.nix { inherit lib; };
+  neovim = import ./neovim.nix { inherit lib; };
+  themes = import ./themes.nix { inherit lib neovim zsh; };
```

```suggestion
  themes = import ./themes.nix { inherit lib neovim; };
```
that's another PR :D